roles/wazuh/ansible-wazuh-manager/tasks/Debian.yml

```diff
@@ -28,6 +28,7 @@
 - name: Debian/Ubuntu | Installing Wazuh repository key
   apt_key:
     url: "{{ wazuh_manager_config.repo.gpg }}"
+    id: "{{ wazuhrepo.key_id }}"
```

Comment:
Please change it to `{{ wazuh_agent_config.repo.key_id }}`

Reply:
in this case it should be {{ wazuh_manager_config.repo.key_id }} even it's the same

Reply:
done